docs/img/.gitattributes

```diff
@@ -0,0 +1,2 @@
+doxygen-docs.png filter=lfs diff=lfs merge=lfs -text
```

Comment:
This file does not have to be added in this PR - we just merged the `git-lfs` support branch so please delete this and merge `develop` again 👍